applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -62,4 +67,32 @@
         </div>
         @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
     }
+
+    <script>
+        require(['Promise'], function (Promise) {
```

Comment:
Users without native promises are a small and dwindling minority – perhaps it should be something like this, to speed things up for the majority?

```js
function init (Promise) {
  // ...
}

window.Promise ? init(Promise) : require(['Promise'], init);
```